lib/widgets/lightbox.dart

```diff
@@ -197,22 +199,207 @@ class _LightboxPageState extends State<_LightboxPage> {
           child: GestureDetector(
             behavior: HitTestBehavior.translucent,
             onTap: _handleTap,
-            child: SizedBox.expand(
-              child: InteractiveViewer(
-                child: SafeArea(
-                  child: LightboxHero(
-                    message: widget.message,
-                    src: widget.src,
-                    child: RealmContentNetworkImage(widget.src, filterQuality: FilterQuality.medium))))))),
-        bottomNavigationBar: bottomAppBar));
+            child: widget.child))));
   }
 }
 
+class _ImageLightboxPage extends StatefulWidget {
+  const _ImageLightboxPage({
+    required this.routeEntranceAnimation,
+    required this.message,
+    required this.src,
+  });
+
+  final Animation routeEntranceAnimation;
+  final Message message;
+  final Uri src;
+
+  @override
+  State<_ImageLightboxPage> createState() => _ImageLightboxPageState();
+}
+
+class _ImageLightboxPageState extends State<_ImageLightboxPage> {
+  @override
+  Widget build(BuildContext context) {
+    return _LightboxPageLayout(
+      routeEntranceAnimation: widget.routeEntranceAnimation,
+      message: widget.message,
+      buildBottomAppBar: (context, color, elevation) => BottomAppBar(
+        color: color,
```

Comment:
This might as well be a method tearoff just like in `VideoLightboxPage`, for maximum parallelism, though it's short enough that it's also fine as a function expression with block body.